content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -164,20 +164,136 @@ view UserInfo {
+
+- The `view` block definition
+- The view's field definitions
+
+These two pieces allow you to define the name of your view in the generated Prisma Client and the columns present in your view's query results.
+
+#### Define a `view` block
+
+To define the `UserInfo` view from the example above, begin by using the `view` keyword to define a `view` block in your schema named `UserInfo`:
+
+```prisma
+view UserInfo {
+  // Fields
+}
+```
+
+<!-- It would be good to reference some recommended naming conventions as we do for `model` blocks -->
+
+#### Define fields
+
+The properties of a view are called _fields_, which consist of:
 
-Currently, Prisma treats views in the same way as models. This means that a view needs to have a [unique identifier](/concepts/components/prisma-schema/data-model#defining-an-id-field). In relational databases, this can be an `@id` or `@unique` attribute on one field, or an `@@id` or `@@unique` attribute on multiple fields. In MongoDB, this must be an `@id` attribute that maps to the `_id` field in the underlying database with `@map("_id")`.
+- A field name
+- A field type
 
-In the example above, the `id` field has an `@id` attribute.
+<!-- Would be good to link off to API reference material for valid attributes on these when available -->
 
-Prisma does not currently support introspection of views with `db pull`. Instead, you must manually add views to your Prisma schema.
+The fields of the `UserInfo` example view can be defined as follows:
+
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+<tab>
+
+```prisma highlight=2-5;normal
+view UserInfo {
+  id    Int    @unique
+  email String
+  name  String
+  bio   String
+}
+```
+
+</tab>
+<tab>
+
+```prisma highlight=2-5;normal
+view UserInfo {
+  id    String  @id @default(auto()) @map("_id") @db.ObjectId
+  email String
+  name  String
+  bio   String
+}
+```
+
+</tab>
+</TabbedContent>
+
+Each _field_ of a `view` block represents a column in the query results of the view in the underlying database.
+
+### Use introspection
+
+If you have an existing view or views defined in your database, [introspection](/concepts/components/introspection) will automatically generate `view` blocks in your Prisma schema that represent those views.
+
+Assuming the example `UserInfo` view exists in your underlying database, running the following command will generate a `view` block in your Prisma schema representing that view:
+
+```sh copy
+npx prisma db pull
+```
+
+The resulting `view` block will be defined as follows:
+
+```prisma
+view UserInfo {
+  id    Int?
+  email String?
+  name  String?
+  bio   String?
+
+  @@ignore
+}
+```
+
+The `view` block is generated initially with a `@@ignore` attribute because [there is no unique identifier defined](#unique-identifier) (which is a currently a [limitation](#unique-identifier) of the views implementation in Prisma).
```

Comment:
```suggestion
The `view` block is generated initially with a `@@ignore` attribute because [there is no unique identifier defined](#unique-identifier) (which is a currently a [limitation](#unique-identifier) of the preview feature views implementation in Prisma).
```